### PR TITLE
[refactor]  flatten the directory structure & unify the name style

### PR DESCRIPTION
**Pull request by PsiACE**
I hereby agree to the terms of the CLA available at: https://datafuse.rs/policies/cla/

## Summary

**Done**

- flatten the directory structure
- unify the name style

**TODO**

- move sql module
- check everywhere, include website

## Changelog

- Build/Testing/CI

## Related Issues

Related #1367 

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

**Comment by PsiACE** [comment]
A dirty looking PR, most of the work has been done automatically. But it needs to be checked carefully.

**Comment by drmingdrmer** [comment]
417 files changed... I do not if my laptop has enough memory to open that tab 🤔

File: docker/cache/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 FROM debian:bullseye
-COPY ./distro/fuse-query /fuse-query
-COPY ./distro/fuse-store /fuse-store
+COPY ./distro/datafuse-query /datafuse-query
```
Comment (ZhiHanZ) [inline docker/cache/Dockerfile:2]: renaming docker binaries should also update test-infra manifests

File: deploy/charts/datafuse/templates/configmap.yaml [hunk 2]
```diff
@@ -5,13 +5,13 @@ metadata:
...
-  FUSE_QUERY_HTTP_API_ADDRESS: "0.0.0.0:{{ .Values.configs.httpAPIPort }}"
-  FUSE_QUERY_METRIC_API_ADDRESS: 0.0.0.0:{{ .Values.configs.metricAPIPort }}
-  FUSE_QUERY_RPC_API_ADDRESS: 0.0.0.0:{{ .Values.configs.rpcAPIPort }}
-  FUSE_QUERY_MYSQL_HANDLER_PORT: {{ .Values.configs.mysqlPort | quote }}
-  FUSE_QUERY_CLICKHOUSE_HANDLER_PORT: {{ .Values.configs.clickhousePort | quote }}
+  QUERY_MYSQL_HANDLER_HOST: "0.0.0.0"
+  QUERY_CLICKHOUSE_HANDLER_HOST: "0.0.0.0"
+  QUERY_HTTP_API_ADDRESS: "0.0.0.0:{{ .Values.configs.httpAPIPort }}"
+  QUERY_METRIC_API_ADDRESS: 0.0.0.0:{{ .Values.configs.metricAPIPort }}
+  QUERY_RPC_API_ADDRESS: 0.0.0.0:{{ .Values.configs.rpcAPIPort }}
```
Comment (ZhiHanZ) [inline deploy/charts/datafuse/templates/configmap.yaml:12]: should be FLIGHT_API_ADDRESS

File: deploy/charts/datafuse/values.yaml [hunk 3]
```diff
@@ -5,7 +5,7 @@
 replicaCount: 1
 
 image:
-  repository: datafuselabs/fuse-query
+  repository: datafuselabs/datafuse-query
```
Comment (ZhiHanZ) [inline deploy/charts/datafuse/values.yaml:8]: helm only supports standalone mode, and standalone image name is datafuselabs/datafuse

**Comment by sundy-li** [comment]
LGTM, it's hard to review the structure changes, if it passed all tests, it can be merged.

File: deploy/charts/datafuse/values.yaml (hunk 3 above)
Comment (BohuTANG) [reply deploy/charts/datafuse/values.yaml:8]: @ZhiHanZ 

If need, you can request a change here.

**Comment by BohuTANG** [comment]
Hi @PsiACE,
A huge patch :), a better way:
1. patch-1: only move 
2. patch-2: rename

File: docker/cache/Dockerfile (hunk 1 above)
Comment (PsiACE) [reply docker/cache/Dockerfile:2]: https://github.com/datafuselabs/test-infra/pull/26

File: deploy/charts/datafuse/templates/configmap.yaml (hunk 2 above)
Comment (PsiACE) [reply deploy/charts/datafuse/templates/configmap.yaml]: done

File: deploy/charts/datafuse/values.yaml (hunk 3 above)
Comment (PsiACE) [reply deploy/charts/datafuse/values.yaml]: done

**Comment by PsiACE** [comment]
> Hi @PsiACE,
> A huge patch :), a better way:
> 
>     1. patch-1: only move
> 
>     2. patch-2: rename

I know, but that would be two huge patches :( 

If it is necessary to split this patch, please let me know.

File: deploy/charts/datafuse/values.yaml (hunk 3 above)
Comment (ZhiHanZ) [reply deploy/charts/datafuse/values.yaml]: agreed, helm needs to be refactored

**Review by BohuTANG — APPROVED** [review]
LGTM

**Pull request merged**